gen/calc.py

```diff
@@ -587,6 +592,7 @@ def validate_exhibitor_storage_master_discovery(master_discovery, exhibitor_stor
         'mesos_dns_ip_sources': '["host", "netinfo"]',
         'master_external_loadbalancer': '',
         'mesos_log_retention_mb': '4000',
+        'mesos_container_log_destination': 'both',
```

Comment:
'both' is the correct default for now, so a) we unblock the infinity team's log-viewing via the Mesos UI, and b) we can test the new journald logging feature.
In the future (1.9-GA? 1.10?) we'd like to move to only journald logging by default. But we have to fix other UI issues first.
Some customers may request disabling the journald task logging (for performance concerns), at which point they can still access task logs in the Mesos/Marathon UIs, or through the DC/OS UI's sandbox viewer. The `dcos task log` CLI may not work properly in this mode? Maybe we should be adjusting the "capabilities" to toggle the CLI's behavior?
cc: @darkonie